klayout/drc/testing/testcases/unit/otp_mk.svg

```diff
@@ -0,0 +1,1530 @@
+<text id="0xfd1e70" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4024.41 5735.18) scale(0.5) scale(1 -1)">Touch</text>
+<text id="0xfd1f20" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4092.29 5736.92) scale(0.5) scale(1 -1)">Space</text>
+<text id="0xfd1fd0" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4008.99 6468.99) scale(1 -1)">Basic</text>
+<text id="0xfd2080" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-2510.73 7219.69) scale(1 -1)">O.CO.7  Space from active contact to poly2 on active = 0.13 (L/MV)</text>
+<text id="0xfd2110" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-3794.9 2831.6) scale(0.5) scale(1 -1)">Corner2Corner</text>
+<text id="0xfd21c0" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-3883.13 2830.1) scale(0.5) scale(1 -1)">Corner2Edge</text>
+<text id="0xfd2270" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-3961.91 2829.73) scale(0.5) scale(1 -1)">Singular</text>
+<text id="0xfd2320" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4196.41 2830.46) scale(0.5) scale(1 -1)">Space</text>
+<text id="0xfd23d0" class="l12t222" text-anchor="start" dominant-baseline="text-after-edge" transform="translate(-4110.99 3560.99) scale(1 -1)">LIMITATION DEPENDENCIES: priWidth &gt; secOvlpPriSide, priWidth&gt;secOvlpPriTopBot,</text>
+<text id="0xfd24c0" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-3982.77 3601.16) scale(1 -1)">Touch</text>
+<text id="0xfd2570" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4123.84 3601.15) scale(1 -1)">Basic</text>
+<text id="0xfd2620" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4054.57 3600.33) scale(1 -1)">Intersect</text>
+<text id="0xfd26d0" class="l12t222" text-anchor="start" dominant-baseline="text-after-edge" transform="translate(-4089.39 3341.02) scale(1 -1)">LIMITATION DEPENDENCIES: priWidth &gt; secOvlpPriSide, priWidth&gt;secOvlpPriTopBot,</text>
+<text id="0xfd27c0" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-3970.77 3381.19) scale(1 -1)">Touch</text>
+<text id="0xfd2870" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4102.24 3381.18) scale(1 -1)">Basic</text>
+<text id="0xfd2920" class="l12t222" text-anchor="start" dominant-baseline="central" transform="translate(-4037.77 3380.36) scale(1 -1)">Intersect</text>
+<text id="0xfd29d0" class="l12t222" text-anchor="start" dominant-baseline="text-after-edge" transform="translate(-4090.09 3324.78) scale(1 -1)">Intersect case will not be checked by DRC because this will fail the LVS.</text>
```

Comment:
can we surface this limitation in the deck itself, or in an issue if that's something we want to track further?

Reply:
@proppy I'm not sure how to deal with this.

Reply:
sorry for the lack for precision, I meant that the comment about the testcase:

> Intersect case will not be checked by DRC because this will fail the LVS

Should probably be surfaced and documented in the deck itself; or as a pending issue if that's something we want to address.

Reply:
@proppy Will check that one.
@FaragElsayed2 Could you please check that case that has that comment?

Reply:
related #56